bin/runRollup.js

```diff
@@ -143,6 +143,16 @@ function bundle ( options ) {
 			return bundle.write( options );
 		}
 
+		if ( options.targets ) {
+			var result = null;
+
+			options.targets.forEach( target =>
+				result = bundle.write( target )
+			);
+
+			return result;
```

Comment:
This really should be done like so:

```js
return Promise.all(
  options.targets.map( target => bundle.write( target ) )
);
```

However, since this is in `bin/` and the `es6-promise` library is bundled and not available as a node dependency, I can't pull it in. What do you think we should do, @Rich-Harris?